cmd/dep/godep_importer_test.go

```diff
@@ -395,3 +338,13 @@ func equalImports(a, b []godepPackage) bool {
 
 	return true
 }
+
+// projectRootPtr takes a ProjectRoot value and returns a pointer.
+func projectRootPtr(p gps.ProjectRoot) *gps.ProjectRoot {
```

Comment:
i'm on the fence - ordinarily, i'd say that we've tried to architect around situations where the zero could have multiple interpretations. but the importers are tricky, because we're converting other systems' semantics into our own, so it's harder to maintain invariants like that. at the same time, it is just a single, confined test pattern, so i don't know that it really matters too much.

i'll leave it up to you, @carolynvs 😄